### DISCUSSION TITLE
Hello! Please add Radio Mirlo station. It is hosted on Zeno-FM :-)

### DISCUSSION THREAD

**Discussion by DragonSerW** [mikepierce/internet-radio-streams#6] [Station Suggestions]
Radio Mirlo official web-site: https://edithsartelli.wixsite.com/radiomirlo/.
Stream (I am not sure, if URLs are right):
https://stream.zeno.fm/e6hg6b5vn7zuv
AND
https://stream-151.zeno.fm/e6hg6b5vn7zuv.

**Comment by mikepierce** [comment]
Thank you for the suggestion :) However I don't see myself regularly listening to this station in the future.
